src/app/services/graph.service.js

```diff
@@ -174,21 +174,9 @@ angular
                     }
                 },
                 {
-                    selector: 'node[resource_type="source"]',
+                    selector: 'node[color]',
                     style: {
-                        'background-color': '#5fb825',
-                    }
-                },
-                {
-                    selector: 'node[resource_type="exposure"]',
-                    style: {
-                        'background-color': '#ff694b',
-                    }
-                },
-                {
-                    selector: 'node[resource_type="metric"]',
-                    style: {
-                        'background-color': '#ff5688',
+                        'background-color': 'data(color)',
```

Comment:
this grabs the value of the `color` property on the node and uses it as the background color on the DAG node

Reply:
Thank you Drew. Literally hours of troubleshooting culminating into this single line of code unblocked my first step. It works! But there's more work to come to ensure node colors work correctly across many types of configs. 

<img width="418" alt="image" src="https://user-images.githubusercontent.com/19176976/172928838-94794880-7daa-481e-a010-8f350e30f1e1.png">